testSrc/unit/io/flutter/sdk/FlutterCreateAddtionalSettingsTest.java

```diff
@@ -0,0 +1,139 @@
+/*
+ * Copyright 2017 The Chromium Authors. All rights reserved.
+ * Use of this source code is governed by a BSD-style license that can be
+ * found in the LICENSE file.
+ */
+package io.flutter.sdk;
+
+import org.junit.Test;
+
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+public class FlutterCreateAddtionalSettingsTest {
+  @Test
+  public void includeDriverPropertyTest() {
+    FlutterCreateAdditionalSettings addtionalSettings1 = new FlutterCreateAdditionalSettings.Builder().setIncludeDriverTest(true).build();
+    FlutterCreateAdditionalSettings addtionalSettings2 = new FlutterCreateAdditionalSettings.Builder().setIncludeDriverTest(false).build();
+    FlutterCreateAdditionalSettings addtionalSettings3 = new FlutterCreateAdditionalSettings.Builder().setIncludeDriverTest(null).build();
+
+    List<String> args1 = addtionalSettings1.getArgs();
+    List<String> args2 = addtionalSettings2.getArgs();
+    List<String> args3 = addtionalSettings3.getArgs();
+
+    assertTrue(args1.size() == 1);
+    assertEquals("--with-driver-test", args1.get(0));
+
+    assertTrue(args2.size() == 0);
+    assertTrue(args3.size() == 0);
+  }
+
+  @Test
+  public void generatePluginPropertyTest() {
+    FlutterCreateAdditionalSettings addtionalSettings1 = new FlutterCreateAdditionalSettings.Builder().setGeneratePlugin(true).build();
+    FlutterCreateAdditionalSettings addtionalSettings2 = new FlutterCreateAdditionalSettings.Builder().setGeneratePlugin(false).build();
+    FlutterCreateAdditionalSettings addtionalSettings3 = new FlutterCreateAdditionalSettings.Builder().setGeneratePlugin(null).build();
+
+    List<String> args1 = addtionalSettings1.getArgs();
+    List<String> args2 = addtionalSettings2.getArgs();
+    List<String> args3 = addtionalSettings3.getArgs();
+
+    assertTrue(args1.size() == 1);
```

Comment:
Not a big deal but failure would read a little better if this were:

    assertEquals(1, args1.size());